omnigibson/objects/object_base.py

```diff
@@ -171,14 +172,6 @@ def _post_load(self):
             PhysxSchema.PhysxArticulationAPI.Apply(root_prim)
             self.self_collisions = self._load_config["self_collisions"]
 
-        # TODO: Do we need to explicitly add all links? or is adding articulation root itself sufficient?
-        # Set the collision group
-        CollisionAPI.add_to_collision_group(
```

Comment:
Why is this removed? I know the actual implementation in `CollisionAPI` is all commented out, but that was because (at the time, which was like pre-CoRL so almost a year ago) omni was buggy and giving us slowdowns + crashes when we were adding a bunch of objects to the collision group.

Reply:
added back